observability/genaisdk/main.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+from dotenv import load_dotenv
+from google.genai import Client
+from otel_setup import setup_otel_instrumentation, setup_otel_to_gcp_wiring
+
+
+# [START main_logic_support_snippet]
```

Comment:
Are you trying to make code snippets that are to be embedded in the documentation? Those type of code samples should be in the python-docs-samples repo. Otherwise, remove the snippet region tags

Reply:
Yes. Is the "python-docs-samples" a technical requirement or a policy one? Any thoughts on this from a discoverability standpoint? (One of the reasons I put this in the "generative-ai" GitHub was in case people started from the repo rather than from the docs).

Reply:
It's more of a policy one, but there are also some technical parts because python-docs-samples has configurations for automatic detection of region tags.

We intend for people to find these kind of code samples from the documentation and sample browser rather than the GitHub Repo.